docs/data/data-grid/row-dimensions/row-dimensions.md

```diff
@@ -0,0 +1,123 @@
+---
+title: Data Grid - Row dimensions
+---
+
+# Data grid - Row dimensions
+
+<p class="description">Customize the dimensions of your rows.</p>
+
+## Static row height
+
+By default, the rows have a height of 52 pixels.
+This matches the normal height in the [Material Design guidelines](https://material.io/components/data-tables).
+
+If you want to create a more / less compact grid and not only set the row height, take a look at our [Density documentation](/x/react-data-grid/accessibility/#density-selector)
+
+To change the row height for the whole grid, set the `rowHeight` prop:
+
+{{"demo": "DenseHeightGrid.js", "bg": "inline"}}
+
+## Variable row height
+
+If you need some rows to have different row heights this can be achieved using the `getRowHeight` prop. This function is called for each visible row and if the return value is a `number` then that `number` will be set as that row's `rowHeight`. If the return value is `null` or `undefined` then the `rowHeight` prop will take effect for the given row.
```

Comment:
```suggestion
If you need some rows to have different row heights this can be achieved using the `getRowHeight` prop. This function is called for each visible row and if the return value is a `number` then that number will be set as that row's `rowHeight`. If the return value is `null` or `undefined`, then the `rowHeight` prop will take effect for the given row.
```